### PR TITLE
Improve SEO and Accessibility score on Lighthouse

### PR DESCRIPTION
**Pull request by fiennyangeln**
-Add aria-label to input and button
-Fix Lists do not contain only `<li>` elements and script supporting
-Fix document does not have meta description

### CODE REVIEW

File: .neutrinorc.js [hunk 1]
```diff
@@ -2,8 +2,14 @@ module.exports = {
   use: [
     ['neutrino-preset-mozilla-frontend-infra/react', {
       html: {
-        title: 'Codetribute'
-      }
+        title: 'Codetribute',
+        meta: [
+          {
+            name: 'Description',
+            content: 'A web app to help people make their first code contribution with Mozilla',
```
Comment (helfi92) [inline .neutrinorc.js]: `content: 'Find your first code contribution with Mozilla',`

File: src/components/TasksTable/index.jsx [hunk 2]
```diff
@@ -295,7 +295,7 @@ export default class TasksTable extends Component {
     return (
       <Fragment>
         <Toolbar className={classes.toolbar}>
-          <Typography variant="title" className={classes.title}>
+          <Typography variant="title" className={classes.title} id="tableTitle">
```
Comment (helfi92) [inline src/components/TasksTable/index.jsx]: Remove `id`.

File: src/components/TasksTable/index.jsx [hunk 3]
```diff
@@ -305,7 +305,7 @@ export default class TasksTable extends Component {
             className={classes.adventurousButton}>
             I’m Feeling Adventurous
           </Button>
-          <IconButton onClick={this.handleFilterToggle}>
+          <IconButton aria-label="Filter" onClick={this.handleFilterToggle}>
```
Comment (helfi92) [inline src/components/TasksTable/index.jsx]: Nice!

File: src/components/TasksTable/index.jsx [hunk 4]
```diff
@@ -360,11 +360,16 @@ export default class TasksTable extends Component {
                 <TableCell className={classes.tableCell}>
                   <IconButton
                     name={item.summary}
+                    aria-label="Info"
```
Comment (helfi92) [inline src/components/TasksTable/index.jsx]: Rename to `Information`.

File: src/components/TasksTable/index.jsx (hunk 2 above)
Comment (fiennyangeln) [reply src/components/TasksTable/index.jsx]: I add id because the table is rendered as `<table aria-labelledby='tableTitle'>`

Comment (helfi92) [reply src/components/TasksTable/index.jsx]: Ahh, ok makes sense. Thanks for clarifying.

**Pull request merged**